packages/components/src/components/chat/QuotaExceededMessage.vue

```diff
@@ -0,0 +1,163 @@
+<template>
+  <div class="message">
+    <span>{{ message }}</span>
+    <div class="options">
+      <div class="option" v-if="limit > 0">
+        <div class="option__grant center">Continue on the free plan</div>
+        <p>The free plan allows you to send up to {{ limit }} messages to Navie AI per day.</p>
+        <p>
+          Please wait <b>{{ timeLeft }} </b>to send another message.
+        </p>
+      </div>
+      <div class="option">
+        <div class="option__grant center">Upgrade to AppMap Pro</div>
+        <a href="https://appmap.io/pricing" class="center" target="_blank">See details</a>
+        <p>AppMap Pro has no daily restriction on the number of daily messages sent to Navie AI.</p>
+        <p>Upgrading will instantly restore access.</p>
+        <a
+          href="https://billing.stripe.com/p/login/14kcOoc0gbrIdqMeUU"
```

Comment:
We should have a page on getappmap.com for this because they may want to choose a personal subscription or a team subscription - or even Enterprise. So maybe “Compare plans”